apps/namadillo/src/App/Common/TransactionReceipt.tsx

```diff
@@ -60,9 +72,17 @@ const TransferTransactionReceipt = ({
       transaction.destinationAddress || ""
     );
   }, [transaction]);
+  const decodedPseudokey = PseudoExtendedKey.decode(
```

Comment:
This can throw. Also I think maybe instead of try catching it's worth adding `can_decode` function to the PseudoExtendedKey struct and just return a boolean